src/content/docs/journey-app-slow/problematic-transactions.mdx

```diff
@@ -0,0 +1,115 @@
+metaDescription: Improve transaction performance with the New Relic transactions UI.
+---
+
+import journeyIntro from 'images/journey_screenshot-intro.webp'
+import journeyConsuming from 'images/journey_screenshot-consuming.webp'
+import journeySegment from 'images/journey_screenshot-segment.webp'
+import journeySlow from 'images/journey_screenshot-slow.webp'
+
+## Objectives [#objectives]
+
+
+
+<SideBySide>
+    <Side>
+By the end of this tutorial, you will:
+
+* Understand transactions and traces
+* Have identified any problematic transactions
+* Reviewed and optimized problematic segments in your app
+    </Side>
+    <Side>
+  <img
+  title="Transaction home page"
+  alt="The transactions home page"
+  src={journeyIntro}
+  />
+    </Side>
+</SideBySide>
+
+
+## The what and why of transactions [#why]
+
+Customer activity on your web app creates routes through your code that begin when an app receives a web request and ends when a response returns. These routes are transactions, or a unit of work made up of function calls and method calls. Think about a critical business point, like a customer submitting payment on a purchase. If a customer reports that their payment either failed to process or processed after significant delay, you’re left guessing about the cause of that failure. Did an API cause the failure, or was it a recent bug fix deployed earlier that day? 
+
+## Observe transactions to improve performance [#observe]
+
+Our transactions UI helps you diagnose the source of a problem so you can improve the performance of specific transactions. The following steps guide you through identifying slow transactions and improving your apps performance:
+
+<Steps>
+    <Step>
+    Navigate to the transactions UI: Go to **[one.newrelic.com](https://one.newrelic.com/nr1-core) > APM & services > (select an app) > Monitor > Transactions**
+    </Step>
+    <Step>
+
+    Find your slow transactions: 
+
+    1. Sort your transactions by most time consuming on the top of the page.
+    2. Take a look at the **Top 20 transactions** chart. This list represents the best opportunity to reduce load of your application. Shaving time off these frequently called transactions will reduce the footprint of your application as they are high volume requests. Take note of the top few transactions in this chart.
+    3. Sort your transactions by slowest average response on the top of the page.
+    4. Take a look at the **Top 20 transactions** chart again. This chart now shows the average time per request from the total volume of a particular request. Take note of the top few transactions in this chart.
+    <SideBySide>
+    <Side>
+  <img
+  title="Most consuming transactions chart"
+  alt="Most consuming transactions chart"
+  src={journeyConsuming}
+  />
+    </Side>
+    <Side>
+  <img
+  title="Slowest transactions chart"
+  alt="Slowest transactions chart"
+  src={journeySlow}
+  />
+</SideBySide>
+    </Step>
+    <Step>
+
+    Improve your performance:
```

Comment:
Having this just as some body text with a colon at the end feels really odd. Suggest either having all steps use h3s, or cut this sentence entirely.